packages/frontend/src/components/chart/configure/controls/onMouseMove.ts

```diff
@@ -0,0 +1,32 @@
+import { Message } from '../messages'
+
+export function onMouseMove(
+  view: HTMLElement,
+  dispatch: (message: Message) => void,
+) {
+  let wasInside = false
+
+  window.addEventListener('mousemove', (e) => onEvent(e))
+  window.addEventListener('touchstart', (e) => onEvent(e.touches[0]))
+  window.addEventListener('touchmove', (e) => onEvent(e.touches[0]))
+
+  function onEvent(e: MouseEvent | Touch) {
+    const rect = view.getBoundingClientRect()
+    const isInside =
+      e.clientX >= rect.left - 16 &&
```

Comment:
`16` seems pretty arbitrary - what's the rationale behind it?

Reply:
It is totally arbitrary. This basically solves for the fact that the first and last value of the chart have 2x less hover space, because every other value has some space to the left and to the right but the edge values only have one of those. So this 16px is compensation.

I can extract it to some named constant.

Reply:
I see, let's give it a name and put a comment with your explanation 👍